lib/jdomodels/src/main/resources/private/dbo-beans.spb.xml

```diff
@@ -21,6 +21,7 @@
 				<bean class="org.sagebionetworks.repo.model.dbo.persistence.DBOGroupMembers" />
 				<bean class="org.sagebionetworks.repo.model.dbo.persistence.DBOCredential" />
 				<bean class="org.sagebionetworks.repo.model.dbo.persistence.DBOSessionToken" />
+				<bean class="org.sagebionetworks.repo.model.dbo.persistence.DBOAuthenticatedOn" />

```

Comment:
This should be added also to the list of beans in the migratableTableDAO bean below (See line 186).